src/main/java/com/ekart/thunderbolt/DAO/ProductDAO.java

```diff
@@ -0,0 +1,50 @@
+package com.ekart.thunderbolt.DAO;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProductDAO {
```

Comment:
DAO stands for Data Access Object, which is generally used for @Entity classes. If this class is used for just data traffic, better suffix would be DTO which stands for Data Transfer Object.